### DISCUSSION TITLE
Additional columns on Call Search screen

### DISCUSSION THREAD

**Discussion by edurxxx** [chuot/rdio-scanner#428] [Ideas]
I would like to see a column for either Call Duration or File Size on the Call Search dialog.
This would be nice to be able to track down the longer dispatch calls that usually include pager tones and a short report of the call vs the quicker and much shorter acknowledgement calls.
I know space on the screen is very limited, but maybe this column could be toggled on/on in the Admin interface somewhere?
Is this something that would be useful to others?

**Comment by flaw101** [comment]
I’ll add my vote for this feature, for the reason stated. Call duration is a good way to filter through dispatches over a period of time.
